app/models/user.rb

```diff
@@ -1,2 +1,4 @@
 class User < ActiveRecord::Base
+  has_many :reviews, as: :reviewable
+  has_many :reviews, foreign_key: :author_id
```

Comment:
naming this association the same as line 2 is going to be problematic. it actually overrides the first association. 

you'll want to name these different things. i'd suggest maybe the following:
```
has_many :reviews, as: :reviewable
has_many :authored_reviews, class_name: "Review", foreign_key: :author_id
```